pkg/cloud/aws/actuators/machine/utils.go

```diff
@@ -141,30 +141,40 @@ func terminateInstances(client awsclient.Client, instances []*ec2.Instance) erro
 func ProviderConfigFromMachine(client client.Client, machine *clusterv1.Machine, codec *providerconfigv1.AWSProviderConfigCodec) (*providerconfigv1.AWSMachineProviderConfig, error) {
 	var providerConfig runtime.RawExtension
 
-	if machine.Spec.ProviderConfig.Value == nil && machine.Spec.ProviderConfig.ValueFrom == nil {
+	// TODO(jchaloup): Remove providerConfig once all consumers migrate to providerSpec
+	providerSpec := machine.Spec.ProviderConfig
+	// providerSpec has higher priority over providerConfig
+	if machine.Spec.ProviderSpec.Value != nil || machine.Spec.ProviderSpec.ValueFrom != nil {
+		providerSpec = machine.Spec.ProviderSpec
+		fmt.Printf("Falling to default providerSpec\n")
+	} else {
+		fmt.Printf("Falling to providerConfig\n")
```

Comment:
Why are we printing as opposed to logging?

Reply:
Leftover for debugging.

Reply:
used `glog.Infof` instead.